### ISSUE TITLE
Are namespaces supported?

### ISSUE THREAD

**Issue by josephbridgwaterrowe** [simukappu/activity_notification#56]
I am having problems adding notifications under an admin namespace.

Some background, the app has a single public page which is used to submit some information and the remaining functionality is used by administrators.

```ruby
# /config/routes.rb
Rails.application.routes.draw do
  # Public routes (a form on the home#index POST's to information#create
  resources :information, only: %i(create)
  root to: 'home#index'

  # Admin section
  namespace :admin do
    devise_for :users, ...

    notify_to :users, controller: '/activity_notification/notifications'

    # Some other information model related routes
    resources :information, only: %i(edit index show update)
  end
end
```

The user and notifiable models are basic:

```ruby
# /app/models/user.rb
class User < ApplicationRecord
  acts_as_notifier
  acts_as_target email: :email_address, email_allowed: true


# /app/models/information.rb
class Information < ApplicationRecord
  acts_as_notifiable :users,
                     email_allowed: true,
                     notifier: ->(user, key) { user },
                     notifiable_path: ->(information, key) { [:admin, information] },
                     tracked: true,
                     targets: -> (information, key) { User.where(role: 'admin').to_a },
                     printable_notifier_name: ->(user, key) { user }
```

However, when visiting http://localhost:5000/admin/users/1/notifications

I receive the following error:

```
undefined method `open_all_user_notifications_path' for #<#<Class...

    # @todo Must handle devise namespace
    def open_all_notifications_path_for(target, params = {})
      send("open_all_#{target.to_resource_name}_notifications_path", target, params)
    end
```

Which I assume is caused by the target being `User` and the view helpers not supporting nesting, unless of course I'm (hopefully) missing something simple.

**Comment by josephbridgwaterrowe** [comment]
It turns out I may have solved this myself after looking over the other open issues, #46 actually gave me the idea to use a scope around the notifications instead.

I'll continue and report my progress.

**Comment by simukappu** [comment]
Hi @josephbridgwaterrowe,

Thank you for your contribution. The activity_notification gem is not currently tested with nested routes or scope. Since the gem uses internal simple funcation to find configured routes, maybe dynamic routing doesn’t work in these conditions.

I would appreciate it if you could give us enhancements to support nested routes or scope.
Thanks!

**Comment by josephbridgwaterrowe** [comment]
@simukappu, thanks for your response.

It turns out I was able to scope it after all, for example:

```
  # config/routes.rb
  scope 'admin' do
    notify_to :users, with_devise: :admin_user, with_subscription: true
  end
```

Thanks again and I will close this issue as I was able to solve the problem with the above code.

**Comment by simukappu** [comment]
Support for routing scope has been released as v1.6.0.
https://rubygems.org/gems/activity_notification/versions/1.6.0
See [CHANGELOG.md](https://github.com/simukappu/activity_notification/blob/master/CHANGELOG.md)
Thanks!
